qt_ui/displayoptions.py

```diff
@@ -58,6 +58,7 @@ class DisplayOptions:
     waypoint_info = DisplayRule("Waypoint Information", True)
     culling = DisplayRule("Display Culling Zones", False)
     flight_paths = FlightPathOptions()
+    actual_frontline_pos = DisplayRule("Display Actual Frontline Location", False)
```

Comment:
Any reason to keep the old option?

Reply:
When the frontline is full width it's rather large on screen.  I changed the display method for debugging, and decided to add the option as a feature switch; if this is preferred we can switch over to it.

Reply:
Actually- there is a reason.  There's still some campaigns that have the frontline charging straight through exclusion zones that are larger than the search for position in `Conflict.frontline_vector`.  In this case, the frontline isn't rendered properly under "show actual frontline".  

Before 2.3 is released, I'll be adding frontline paths to all the default campaigns and fine-tuning the landmaps so that impossible conflict locations don't occur.